src/map/atcommand.cpp

```diff
@@ -8368,31 +8368,34 @@ ACMD_FUNC(whodrops)
 		sprintf(atcmd_output, msg_txt(sd,1285), item_db.create_item_link( id ).c_str(), id->nameid); // Item: '%s' (ID:%u)
 		clif_displaymessage(fd, atcmd_output);
 
-		if (id->mob[0].chance == 0) {
+		if (id->mobs.empty()) {
 			strcpy(atcmd_output, msg_txt(sd,1286)); //  - Item is not dropped by mobs.
 			clif_displaymessage(fd, atcmd_output);
-		} else {
-			sprintf(atcmd_output, msg_txt(sd,1287), MAX_SEARCH); //  - Common mobs with highest drop chance (only max %d are listed):
-			clif_displaymessage(fd, atcmd_output);
+			return 0;
```

Comment:
```suggestion
			continue;
```
It can be more than one item, so sadly we cannot cancel early.

Reply:
oops, you're right